aten/src/ATen/core/Formatting.cpp

```diff
@@ -292,6 +292,11 @@ std::ostream& print(std::ostream& stream, const Tensor & tensor_, int64_t linesi
         stream << ", axis: " << tensor_.q_per_channel_axis();
       }
     }
+
+    auto& fw_grad = tensor.fw_grad(/* level */ 0);
+    if (fw_grad.defined()) {
+      stream << ", tangent:" << std::endl << fw_grad;
```

Comment:
If we expose this as "tangent" in the user interface, I wonder why we didn't just call it `tensor.tangent()` lol (no action needed)

Reply:
For now the only user interface to extract this information is `fwAD.unpack_dual()`.
But we can consider a user facing method in the future for this but I am not sure we actually need it.